register new envs

dion-jy opened the pull request:
the original repo has some new gym environments for visualizing subgoals.
but we have to replace directly gym files in the installation directory to utilize the new environments.

I think we can register those environments separately. so I changed a little bit.
this PR includes some codes to register environments and to test.

thanks.

dion-jy committed:
register new envs

nikhilbarhate99 commented:
Hi, Thanks for the PR.

Is the `max_episode_steps=50` while registering the env same as that of the original env?

and are both env rendering properly? 
because I had added some extra functions in the `rendering.py` file, but in the new envs that are registered you are still importing the rendering file from the original gym directory, `from gym.envs.classic_control import rendering` (I think this will need to be changed to import the new modified `rendering.py` in the assets). It will still render properly if you have already replaced the original files on your pc, so it is importing the new modified file.

please test your new code with original gym files (i.e. they are not the replaced ones from my repo) and let me know if it works.

dion-jy commented:
> Is the max_episode_steps=50 while registering the env same as that of the original env?

Oh, I forgot to remove this. that was just for the test. I updated.

> please test your new code with original gym files

I didn't change the importing path on env scripts, but it works. 
Anyway, as you modified something on `rendering.py`, I updated it.
Thanks,  give me feedback freely!

dion-jy committed:
update after feedback

dion-jy committed:
importing path fixed

dion-jy committed:
update requirements
